docs/en/observability/logs-stream.asciidoc

```diff
@@ -28,75 +28,62 @@ On your host, download and extract the installation package that corresponds wit
 include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/download-widget.asciidoc[]
 
 [discrete]
-[[logs-stream-agent-config]]
-== Step 2: Configure the {agent}
+[[logs-stream-install-agent]]
+== Step 2: Install and start the {agent}
+After downloading and extracting the installation package, you're ready to install the {agent}. From the agent directory, run the the command that corresponds with your system to install the {agent} as a service. Do *not* enroll the agent in Fleet.
```

Comment:
You mention here "Do not enroll ...". As Fleet and enroll is not part of the guide, is there a chance the user would do this? If not, this introduces new terms the user does not even know about so would rather leave it out.

Reply:
You're prompted to enroll in Fleet when installing, so they will be introduced to it there.  I can clarify to say "Do not  enroll the agent in Fleet when prompted during installation" or ignore it if you think a user might be familiar enough with the concepts of Fleet and standalone agents at this point.

Reply:
I doubt a user is familiar is enough and shouldn't be. ++ on having it and potentially get this step out of the way.